src/CommonAccessors.jl

```diff
@@ -0,0 +1,108 @@
+# Common get and set methods
```

Comment:
I kind of like the idea of the get methods by the types. It makes it easier to find. But doc strings will also help so I we move it here #233 becomes more important for me.

Reply:
> get methods by the types

Sorry @Affie, what do you mean by that?  I think you did see #232 right?

Reply:
Sorry wasn’t very clear. I mean for example  getTags(v::DFGvariable) in DFGVariable.jl by the struct. I thank thats where it came from. If all get methods are just get+fieldname then i guess it will be easy to find.

Reply:
thanks, I updated #232 accordingly

Reply:
Personally I like gets and sets (Java style) - think we all agree on that? A couple are not that way and they should be cleaned up.